charts/argocd-image-updater/templates/configmap.yaml

```diff
@@ -31,6 +31,12 @@ data:
   git.commit-message-template: |
     {{- nindent 4 . }}
   {{- end }}
+  {{- with .Values.config.gitCommitSigningKey }}
```

Comment:
Do we also need somthing like `config.gitConfigSigningMethod`?

Reply:
ahhhh you are right 😇

Reply:
I fixed in https://github.com/argoproj/argo-helm/pull/2809/commits/e64396a40cd257884a7fd9ec9ed52d38887a9669 . 🙋